packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -239,47 +269,69 @@ class _CupertinoButtonState extends State<CupertinoButton> with SingleTickerProv
         ? primaryColor
         : CupertinoDynamicColor.resolve(CupertinoColors.placeholderText, context);
 
+    final Color effectiveFocusOutlineColor = widget.focusColor ??
+      HSLColor
+        .fromColor((backgroundColor ?? CupertinoColors.activeBlue).withOpacity(0.80))
+        .withLightness(0.69).withSaturation(0.835)
```

Comment:
```suggestion
        .withLightness(0.69).withSaturation(0.835)
```

We should also document what's happening here in the `focusColor` property docs.